components/esb-tools/plugins/org.wso2.integrationstudio.esb.form.editors/src/org/wso2/integrationstudio/esb/form/editors/ui/widgets/CompositeElements.java

```diff
@@ -0,0 +1,400 @@
+     * 
+     * @param id Field name of the property
+     * @param widgetFactory widgetFactory instance
+     * @param parent parent composite
+     * @param columns number of columns of the field
+     * @param span span of the field
+     * @return composite
+     */
+    public static Composite createComposite(FormToolkit widgetFactory, Composite parent, int columns, int span) {
+        Composite composite = widgetFactory.createComposite(parent, SWT.TRANSPARENT);
+        composite.setBackground(new Color(parent.getShell().getDisplay(), 255, 255, 255));
+        GridLayout propertiesGroupLayout = new GridLayout();
+        propertiesGroupLayout.numColumns = columns;
+        propertiesGroupLayout.marginLeft = 0;
+        propertiesGroupLayout.horizontalSpacing = 0;
+        propertiesGroupLayout.verticalSpacing = 0;
+        propertiesGroupLayout.marginHeight = 1;
+        propertiesGroupLayout.marginWidth = 1;
+        composite.setLayout(propertiesGroupLayout);
+        GridData propertiesSectionData = new GridData(GridData.FILL_HORIZONTAL);
+        propertiesSectionData.horizontalSpan = span;
+        composite.setLayoutData(propertiesSectionData);
+        return composite;
+
+    }
+
+    public static String getIconPath(String iconName) throws URISyntaxException, IOException {
+        Bundle bundle = Platform.getBundle(PLUGIN_ID);
+        URL webAppURL = bundle.getEntry(iconName);
+        URL resolvedFolderURL = FileLocator.toFileURL(webAppURL);
+        URI resolvedFolderURI = new URI(resolvedFolderURL.getProtocol(), resolvedFolderURL.getPath(), null);
+        File resolvedWebAppFolder = new File(resolvedFolderURI);
+        return resolvedWebAppFolder.getAbsolutePath();
+    }
+
+    /**
+     * Opens namespaced property editor for expression widgets
+     * 
+     * @param parent parent composite
+     * @param valueExpressionText expression text box
+     * @param valueExpression namespaced property instance
+     * @return isSaved
+     */
+    public static boolean openValueExpressionWidgetNamespacedPropertyEditor(final Composite parent,
+            NamespacedProperty valueExpression) {
+        if (valueExpression == null) {
+            valueExpression = EsbFactoryImpl.eINSTANCE.createNamespacedProperty();
+        }
+        ValueExpressionWidgetNameSpacedPropertyEditorDialog nspd = new ValueExpressionWidgetNameSpacedPropertyEditorDialog(
+                parent.getShell(), SWT.NULL, valueExpression);
+        valueExpression = nspd.open();
+        return nspd.isSaved();
+    }
+
+    public static GridData createPropertiesTableComposition(FormToolkit widgetFactory, Composite parent,
+            List<ViewerFilter> propertiesFilters) {
+
+        ReferencesTable properties = new ReferencesTable("Sample label to display", new ReferencesTableListener() {
+            public void handleAdd() {
+                // propertiesEditionComponent.firePropertiesChanged(new
```

Comment:
Let's remove commented sections

Reply:
Fixed in https://github.com/wso2/integration-studio/pull/1114